lib/hexpm_web/views/api/user_view.ex

```diff
@@ -17,6 +17,10 @@ defmodule HexpmWeb.API.UserView do
     render_one(user, __MODULE__, "minimal")
   end
 
+  def render("audit_logs." <> _, %{audit_logs: audit_logs}) do
+    render_many(audit_logs, HexpmWeb.API.AuditLogView, "show")
```

Comment:
Note:
I made `AuditLogView` a separate view in its self. 
So that we can reuse it in `UserView`, `PackageView`, and `OrganizationView`